### ISSUE TITLE
Add "dynamic" rules on the fly for taxes calculation

### ISSUE THREAD

**Issue by dlresende** [dlresende/extreme-carpaccio#52]
Today the problem is very static and sometimes people tend to implement it "from one shot". Furthermore, this approach and the /feedback resources don't push participants to test.

After many feedbacks and discussions with others running Extreme Carpaccio, we're willing to add new taxes rules on the fly to push the problem nature from static (at the beginning) to dynamic (during the game).

To do so, at some point we'd like to change the taxes strategy leaving the fixed numbers back and start using rules for each country, like : `if(total < 100) return sum * 1.2; else return sum * 1.2 + 10;`

**Comment by Arnauld** [comment]
in progress

**Comment by dlresende** [comment]
@Arnauld: it seems the following test case is failing:
```
  1) Countries should keep tax unchanged when it fails to execute evaluation from configuration
   Message:
     Expected 7 to be 281.82.
```
What do you mean by _fails to execute evaluation from configuration_? Is that when the syntax is not correct?

**Comment by dlresende** [comment]
Fixed by #60
